etstool.py

```diff
@@ -107,6 +107,7 @@
     "nose",
     "coverage",
     "configobj",
+    "docutils"
```

Comment:
should it be here or under extra_dependencies?

Reply:
`extra_dependencies` is being used to get dependencies for a particular toolkit. Also, other optional dependencies are listed under `dependencies`.